database/things/rexx.pldb

```diff
@@ -136,6 +136,9 @@ rijuRepl https://riju.codes/rexx
   
 tryItOnline rexx
 
+indeedJobs rexx engineer
+ 2022 325  
```

Comment:
Again, I'm surprised that the number is so high. Are these jobs at IBM?

Reply:
true, I think this should come to 40. 
Thanks for the feedback @celtic-coder  let me revise these values

Reply:
not all jobs are at IBM. [link](https://www.indeed.com/jobs?q=rexx&l=Texas&vjk=5341201a75293683)